src/cli/utils/testdata/flags.go

```diff
@@ -0,0 +1,46 @@
+package testdata
+
+import "strings"
+
+func FlgInpts(in []string) string { return strings.Join(in, ",") }
+
+var (
+	BackupInpt = "backup-id"
+
+	UsersInpt  = []string{"users1", "users2"}
```

Comment:
nit: Should we use Input instead of inpt everywhere? Improves readability.

Reply:
Sure, why not.